pcsx2/GS.cpp

```diff
@@ -138,13 +139,24 @@ __fi void gsWrite8(u32 mem, u8 value)
 	GIF_LOG("GS write 8 at %8.8lx with data %8.8lx", mem, value);
 }
 
+static void GetColorBurst(u32 mem, u32 value)
+{
+	if (mem != GS_SMODE1)
+		return;
+
+	GSRegSMODE1 Register;
+	Register.SMODE1 = value;
+	if(u32 colorburst = Register.CMOD)
```

Comment:
I don't like the mix of condition and affection. By the way, you can easily write
if (CMOD == 3)
 ... PAL
else if (CMOD != 0)
... NTSC